### DISCUSSION TITLE
group-hover:${sliderClassName}

### DISCUSSION THREAD

**Discussion by elevenwangyang** [tailwindlabs/tailwindcss#11740] [Help]
What version of Tailwind CSS are you using?

3.3.3

What build tool (or framework if it abstracts the build tool) are you using?

Next.js 13.4.12

What version of Node.js are you using?

v16.16.0

What browser are you using?

Chrome

What operating system are you using?

macOS

![image](https://github.com/tailwindlabs/tailwindcss/assets/24545670/715a3009-8f65-42ab-ae3c-7cac199aa9fd)

**Comment by RobinMalfait** [comment]
Hey!

Yep, dynamic class names are not supported like that: https://tailwindcss.com/docs/content-configuration#dynamic-class-names

Make the `group-hover:` part of the `sliderClassName` and it should work as expected.

**Reply by elevenwangyang** [reply]
thanks
